### PR TITLE
feat: curve2poolswap address and router address

### PR DESCRIPTION
**Pull request by Aercwarden**

### CODE REVIEW

File: src/abi/Curve2PoolSwap.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      "name": "USDC",
+      "outputs": [
+        {
+          "internalType": "contract IERC20",
+          "name": "",
+          "type": "address"
+        }
+      ],
+      "stateMutability": "view",
+      "type": "function"
```
Comment (witherblock) [inline src/abi/Curve2PoolSwap.json]: is this needed?

File: src/addresses/index.ts [hunk 2]
```diff
@@ -147,6 +147,7 @@ const Addresses: { [key: number]: any } = {
     Tzwap1inchRouter: '0xDA615781745fe52Fc3383Ad23cE9184419bEec30',
     Escrow: '0x6D9c288708a77d51Fa66393916511Cde60522baF',
     CurveGaugesOracle: '0xF45f20bc21C9933ae8613EdD8EF108b7fD25E527',
+    Curve2PoolSwap: '0x07512aF4a8720a04226D2E9c6d802AA25da8D921',
```
Comment (witherblock) [inline src/addresses/index.ts]: is this needed?

**Review by witherblock — CHANGES_REQUESTED** [review]

File: src/abi/Curve2PoolSwap.json (hunk 1 above)
Comment (Aercwarden) [reply src/abi/Curve2PoolSwap.json]: no, will remove

File: src/addresses/index.ts (hunk 2 above)
Comment (Aercwarden) [reply src/addresses/index.ts:150]: no, will remove

**Review by witherblock — APPROVED** [review]

**Pull request merged**